enzyme/Enzyme/ActivityAnalysis.cpp

```diff
@@ -313,7 +313,9 @@ const std::set<Intrinsic::ID> KnownInactiveIntrinsics = {
     Intrinsic::roundeven,
     Intrinsic::lround,
     Intrinsic::llround,
+#if LLVM_VERSION_MAJOR <= 20
     Intrinsic::nvvm_barrier0,
```

Comment:
It’s important we handle the new intrinsic, was that already in the list

Reply:
This is equivalent to cuda sync threads for contact

Reply:
from the PR `nvvm_barrier_cta_sync_aligned_all` seems to be the one?

Reply:
Yeah I think so